README.md

```diff
@@ -12,7 +12,7 @@ List of projects that provide terminal user interfaces
 - [Miscellaneous](#miscellaneous)
 - [Multimedia](#multimedia)
 - [Productivity](#productivity)
-- [Web](#web)
+- [Web](#web)e
```

Comment:
extra `e`

Reply:
don't know how I missed that haha! resolved!